python/singa/autograd.py

```diff
@@ -583,6 +583,107 @@ def backward(self, dy):
 def flatten(x):
     return Flatten()(x)[0]
 
+class Conv2d_GPU(Operation):
```

Comment:
there is no Conv2d_CPU..

Reply:
i will implement it later or merge cpu and gpu part.